patch_migrations.py

```diff
@@ -0,0 +1,154 @@
+import sys
+import sqlite3
+from contextlib import contextmanager
+
+blocks_columns = [
+	'hash',
+	'parentHash',
+	'uncleHash',
+	'coinbase',
+	'root',
+	'txRoot',
+	'receiptRoot',
+	'mixDigest',
+	'td',
+	'baseFee',
+	'withdrawalHash',
+]
+
+withdrawals_columns = [
+	'address',
+	'blockHash'
+]
+
+transactions_columns = [
+	'hash',
+	'r',
+	's',
+	'sender',
+	'func'
+]
+
+event_logs_colums = [
+	'address',
+	'transactionHash',
+	# 'transactionIndex', transaction index needs a different fix
```

Comment:
It does need a different fix, which I don't see here. I did want to note, however, that transactionIndex has fewer than 900 possible values, and you might be able to get away with something like:

```
UPDATE event_logs SET transactionIndex = 1 WHERE transactionIndex = X'0000000100000000000000000000000000000000000000000000000000000000';
UPDATE event_logs SET transactionIndex = 2 WHERE transactionIndex = X'0000000200000000000000000000000000000000000000000000000000000000';
...
```

for each of the ~900 possible values